bcda/utils/common.go

```diff
@@ -79,3 +79,25 @@ func CloseFileAndLogError(f *os.File) {
 		logrus.Error(ferr)
 	}
 }
+
+// This is function that takes a string slice and removes any duplicates
+func Dedup(slice []string) []string {
+	// Get the length of the slice
+	var n = len(slice)
+
+	// Use the length to allocate memory once for new slice and map
+	var newslice = make([]string, 0, n)
```

Comment:
Can we update these variables so that we use MixedCaps? Reference: https://golang.org/doc/effective_go.html#mixed-caps